### PR TITLE
Fix the return_abrupt problem in typedarray_prototype_join

### PR DESCRIPTION
**Pull request by orkvi**
JerryScript-DCO-1.0-Signed-off-by: Virag Orkenyi orkvi@inf.u-szeged.hu

### CODE REVIEW

**Review by zherczeg — APPROVED** [review]
LGTM

File: jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c [hunk 1]
```diff
@@ -798,68 +798,60 @@ ecma_builtin_typedarray_prototype_join (ecma_object_t *obj_p, /**< this object *
     return ECMA_VALUE_ERROR;
   }
 
-  ecma_value_t ret_value = ECMA_VALUE_ERROR;
-
   /* 3. */
   uint32_t length = ecma_number_to_uint32 (length_number);
```
Comment (galpeter) [inline jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c:802]: If I understand the code correctly we could also free the `length_value` here. After this point it is only released and not used in any other ways.

**Review by galpeter — CHANGES_REQUESTED** [review]
A minor change request.

**Review by galpeter — APPROVED** [review]
lgtm

**Comment by rerobika** [comment]
please rebase

**Pull request closed without merging**